doc/guide/go_quickstart.hpp

```diff
@@ -0,0 +1,230 @@
+gives a few examples of usage, and pointers to deeper documentation.
+
+This quickstart guide is also available for
+@ref cli_quickstart "the command-line", @ref python_quickstart "Python"
+and @ref julia_quickstart "Julia".
+
+@section go_quickstart_install Installing mlpack
+
+Installing the mlpack bindings for Go is somewhat time-consuming as the library
+must be built; you can run the following code:
+
+@code{.sh}
+go get -d mlpack.org/v1/mlpack
+cd ${GOPATH}/src/mlpack.org/v1/mlpack
+make install
+@endcode
+
+Building the Go bindings from scratch is a little more in-depth, though.  For
+information on that, follow the instructions on the @ref build page, and be sure
+to specify @c -DBUILD_GO_BINDINGS=ON to CMake;
+
+@section go_quickstart_example Simple mlpack quickstart example
+
+As a really simple example of how to use mlpack from Go, let's do some
+simple classification on a subset of the standard machine learning @c covertype
+dataset.  We'll first split the dataset into a training set and a testing set,
+then we'll train an mlpack random forest on the training data, and finally we'll
+print the accuracy of the random forest on the test dataset.
+
+You can copy-paste this code directly into main.go to run it.
+@code{.go}
+package main
+
+import (
+  "github.com/mlpack.org/v1/mlpack"
+  "fmt"
+)
+func main() {
+
+  // Download dataset.
+  mlpack.DownloadFile("https://www.mlpack.org/datasets/covertype-small.data.csv.gz",
+                      "data.csv.gz")
+  mlpack.DownloadFile("https://www.mlpack.org/datasets/covertype-small.labels.csv.gz",
+                      "labels.csv.gz")
+
+  // Extract/Unzip the dataset.
+  mlpack.UnZip("data.csv.gz", "data.csv")
+  dataset, _ := mlpack.Load("data.csv")
+
+  mlpack.UnZip("labels.csv.gz", "labels.csv")
+  labels, _ := mlpack.Load("labels.csv")
+
+  // Split the dataset using mlpack.
+  params := mlpack.PreprocessSplitOptions()
+  params.InputLabels = labels
+  params.TestRatio = 0.3
+  params.Verbose = true
+  test, test_labels, train, train_labels :=
+      mlpack.PreprocessSplit(dataset, params)
+
+  // Train a random forest.
+  rf_params := mlpack.RandomForestOptions()
+  rf_params.NumTrees = 10
+  rf_params.MinimumLeafSize = 3
+  rf_params.PrintTrainingAccuracy = true
+  rf_params.Training = train
+  rf_params.Labels = train_labels
+  rf_params.Verbose = true
+  rf_model, _, _ := mlpack.RandomForest(rf_params)
+
+  // Predict the labels of the test points.
+  rf_params_2 := mlpack.RandomForestOptions()
+  rf_params_2.Test = test
+  rf_params_2.InputModel = &rf_model
+  rf_params_2.Verbose = true
+  _, predictions, _ := mlpack.RandomForest(rf_params_2)
+
+  // Now print the accuracy.
+  rows, _ := predictions.Dims()
+  var sum int = 0
+  for i := 0; i < rows; i++ {
+    if (predictions.At(i, 0) == test_labels.At(i, 0)) {
+      sum = sum + 1
+    }
+  }
+  fmt.Println(sum, " correct of ", rows, "(", (float64(sum) / float64(rows)) *
+              100, "%)")
```

Comment:
```suggestion
  fmt.Print(sum, " correct out of ", rows, " (",
      (float64(sum) / float64(rows)) * 100, "%).\n")
```
I think this might work better for formatting.  The current code gives this as output:
```
24542  correct of  30000 ( 81.80666666666667 %)
```
but with the suggestion the output is:
```
24669 correct out of 30000 (82.23%).
```

Reply:
Commit the suggestion. :)